apps/main/src/lend/components/PageLoanManage/ManageLoanTabs.tsx

```diff
@@ -0,0 +1,103 @@
+import LoanBorrowMore, { LoanBorrowMoreWrapped } from '@/lend/components/PageLoanManage/LoanBorrowMore'
+import LoanCollateralAdd, { LoanAddCollateralTab } from '@/lend/components/PageLoanManage/LoanCollateralAdd'
+import LoanCollateralRemove, { LoanRemoveCollateralTab } from '@/lend/components/PageLoanManage/LoanCollateralRemove'
+import LoanRepay, {
+  LoanRepayFromCollateralTab,
+  LoanRepayFromWalletTab,
+} from '@/lend/components/PageLoanManage/LoanRepay'
+import LoanSelfLiquidation from '@/lend/components/PageLoanManage/LoanSelfLiquidation'
+import networks from '@/lend/networks'
+import { type MarketUrlParams, PageContentProps } from '@/lend/types/lend.types'
+import { useClosePositionTab } from '@/llamalend/features/manage-soft-liquidation/hooks/useClosePositionTab'
+import { useImproveHealthTab } from '@/llamalend/features/manage-soft-liquidation/hooks/useImproveHealthTab'
+import { ClosePosition } from '@/llamalend/features/manage-soft-liquidation/ui/tabs/ClosePosition'
+import { ImproveHealth } from '@/llamalend/features/manage-soft-liquidation/ui/tabs/ImproveHealth'
+import { useManageLoanMuiForm, useManageSoftLiquidation } from '@ui-kit/hooks/useFeatureFlags'
+import { t } from '@ui-kit/lib/i18n'
+import { type FormTab, FormTabs } from '@ui-kit/shared/ui/FormTabs/FormTabs'
+
+type ManageLoanProps = PageContentProps<MarketUrlParams>
+
+export const ImproveHealthTab = ({ rChainId, rOwmId }: ManageLoanProps) => (
+  <ImproveHealth {...useImproveHealthTab({ chainId: rChainId, network: networks[rChainId], marketId: rOwmId })} />
+)
+
+export const ClosePositionTab = ({ rChainId, rOwmId }: ManageLoanProps) => (
+  <ClosePosition {...useClosePositionTab({ chainId: rChainId, network: networks[rChainId], marketId: rOwmId })} />
+)
+
+export const LendManageLegacyMenu = [
+  {
+    value: 'loan',
+    label: t`Borrow`,
+    subTabs: [
+      { value: 'loan-increase', label: t`Borrow more`, component: LoanBorrowMore },
+      { value: 'loan-decrease', label: t`Repay`, component: LoanRepay },
+      { value: 'loan-liquidate', label: t`Self-liquidate`, component: LoanSelfLiquidation },
+    ],
+  },
+  {
+    value: 'collateral',
+    label: t`Borrow`,
+    subTabs: [
+      { value: 'collateral-increase', label: t`Add collateral`, component: LoanCollateralAdd },
+      { value: 'collateral-decrease', label: t`Remove collateral`, component: LoanCollateralRemove },
+    ],
+  },
+  {
+    value: 'leverage',
+    label: t`Leverage`,
+    visible: ({ market }) => market?.leverage?.hasLeverage(),
+    component: LoanBorrowMore,
+  },
+] satisfies FormTab<ManageLoanProps>[]
+
+export const LendManageNewMenu = [
+  {
+    value: 'borrow',
+    label: ({ market }) => (market?.leverage?.hasLeverage() ? t`Leverage` : t`Borrow`),
+    component: LoanBorrowMoreWrapped,
+  },
+  {
+    value: 'repay',
+    label: t`Repay`,
+    subTabs: [
+      { value: 'from-wallet', label: t`From wallet`, component: LoanRepayFromWalletTab },
```

Comment:
Now that I think of it, all these tab components are only used once and it's directly here. Wouldn't mind inlining them either. I think it's nice to have it all consolidated in the same file instead of being fragmented